README.md

```diff
@@ -5,10 +5,27 @@ Zotero Translation Server + Simple Scraping API + Swagger in Docker
 
 <!-- BEGIN-MARKDOWN-TOC -->
 * [Introduction](#introduction)
-* [Deployment](#deployment)
+* [Installation](#installation)
 * [Configuration](#configuration)
```

Comment:
The `README.md` is getting quite lengthy. You could put the configuration in detail in another file `CONFIGURATION.md` and link to it from here.